fiasco/ion.py

```diff
@@ -416,7 +416,7 @@ def emissivity(self, density: u.cm**(-3), **kwargs) -> u.erg * u.cm**(-3) / u.s:
         contribution_function : Calculate contribution function, :math:`G(n,T)`
         """
         g = self.contribution_function(density, **kwargs)
-        return g * (density**2)[np.newaxis, :, np.newaxis]
```

Comment:
I'm not sure what these `newaxis` bits were doing here?

Reply:
Hmm I think because `g` has dimensions (`ntemperture`, `ndensity`, `ntransitions`)? I'm confused as to how this works now that you've removed those `newaxis` calls. Maybe I just don't understand numpy broadcasting at all? My understanding was that in order to do broadcast like this, the trailing dimension of each array had to have the same length.

Reply:
Yep, you're right, just needed a `atleast_1d` in case density is a scalar.